test/integration/user-management-api/services/proxies_test.rb

```diff
@@ -43,13 +43,6 @@ def test_crud_provider_key
   def test_update
     params = provider_key_params.merge(proxy: { endpoint: 'https://alaska.wild' })
 
-    Proxy.update_all(apicast_configuration_driven: false)
-
-    assert_no_change of: ProxyConfig.method(:count) do
-      put(admin_api_service_proxy_path(params))
-      assert_response :success
-    end
```

Comment:
This doesn't look as a V1 specific test but might be service mesh or other integration. Could you explain why test failed?

Reply:
Agree with @akostadinov , I don't think we have to remove this.

Reply:
Right, this can be service_mesh